pkg/server/sandbox_run.go

```diff
@@ -456,9 +456,21 @@ func (c *criService) generateSandboxContainerSpec(id string, config *runtime.Pod
 	return g.Config, nil
 }
 
-// setupSandboxFiles sets up necessary sandbox files including /dev/shm, /etc/hosts
-// and /etc/resolv.conf.
+// setupSandboxFiles sets up necessary sandbox files including /dev/shm, /etc/hosts,
+// /etc/resolv.conf and /etc/hostname.
```

Comment:
we should probably start flagging *nix for content not applicable to other platforms..

Reply:
We'll do that when the windows support is in.